johngorithm/github-ap-with-java
# #155965831 display a list view of github users

#### What does this PR do?
- Display a list view of github users

#### Description of tasks to be completed
- add required dependencies
- remove dummy user card view
- create a view adapter
- instantiate and connect the adapter class to the main activity
- display users' data with the recycler view
- fix `linter` issues

#### How should this be manually tested?
- clone this repository
- open the project directory using `Android Studio`
- checkout into `ft-display-list-of-github-users-155965831` from Android studio integrated command line
- have an `emulator` downloaded and setup in your IDE
- run the app with `Run` button provided by the IDE and select an emulator
- list of GitHub java developers in Nairobi should be displayed and recycled on scroll

#### Any background context you want to provide?
No

#### Screenshots
<img width="1440" alt="screen shot 2018-12-10 at 9 49 14 am" src="https://user-images.githubusercontent.com/30831276/49721172-ed6a6400-fc61-11e8-895f-bea03f9fa2a0.png">

<img width="1440" alt="screen shot 2018-12-10 at 9 48 21 am" src="https://user-images.githubusercontent.com/30831276/49721209-0115ca80-fc62-11e8-9674-2c8e7554e568.png">




#### What are the relevant Pivotal Tracker Stories?
[155965831](https://www.pivotaltracker.com/story/show/155965831)
[155965773](https://www.pivotaltracker.com/story/show/155965773)
[155903085](https://www.pivotaltracker.com/story/show/155903085)
